giraffe/src/components/Gauge.tsx

```diff
@@ -171,8 +171,17 @@ const drawSegmentedGauge = (
   colors: Color[],
   gaugeSize: number
 ): void => {
+  const thresholdColors = []
+  if (Array.isArray(colors)) {
+    colors.forEach(color => {
```

Comment:
This looks like `filter` rather than `forEach`

Reply:
i agree